packages/extractor-helpers/src/job-helper.js

```diff
@@ -10,8 +10,8 @@
  * governing permissions and limitations under the License.
  */
 
-import { RestError } from '@adobe/content-lake-commons';
 import { randomUUID } from 'crypto';
+import { RestError } from '@adobe/content-lake-commons';
```

Comment:
i can't wait for the day we move this to contentlake-shared :)

Reply:
Yeah, I'd be nice to pull the remaining stuff out of commons/extractors-shared and deprecate both.